index.html

```diff
@@ -0,0 +1,1356 @@
+                            </div>
+                            <div class="story-img">
+                                <a href="#">
+                                    <img src="images/mcconnell-stimulus.jpg" alt="crowded street">
+                                </a>
+                            </div>
+                        </div>
+                        <div class="story-content">
+                            <h2><a href="#">New Stimulus Could Come in 'Next Month or So,' McConnell Says</a></h2>
+                            <div class="summary">
+                                <p>The Kentucky Republican has said in recent weeks that he is eyeing another stimulus worth around $1 trillion, which is unlikely to include another round of individual checks for Americans.</p>
+                            </div>
+                        </div>
+                    </article>
+                    <article class="story col-md-6 col-lg-3">
+                        <div class="story-img-container">
+                            <div class="story-img-category">
+                                <a href="#">u.s.</a>
+                            </div>
+                            <div class="story-img">
+                                <a href="#">
+                                    <img src="images/mall-businesses.jpg" alt="crowded street">
+                                </a>
+                            </div>
+                        </div>
+                        <div class="story-content">
+                            <h2><a href="#">Stimulus Plan to Keep Workers on Payroll Gains Traction in Congress </a></h2>
+                            <div class="summary">
+                                <p>The HEROES Act would not only provide another round of $1,200 stimulus checks, and raises for essential workers such as first responders and physicians, but it also includes an expanded wage subsidy designed to help businesses keep their employees on the payroll.</p>
+                            </div>
+                        </div>
+                    </article>
+                </div>
+            </section>
+            <div class="last-bottom-pick-section">
+                <div class="last-bottom-pick">
+                  <div class="block-title no-border">
+                    <a href="#">U.S.</a>
+                  </div>
+                  <div class="row">
+                    <div class="col-md-4">
+                      <article>
+                        <div class="article-wrap">
+                          <div class="featured-article-img url1"></div>
+                        </div>
+                        <h4>
+                          <a href="#">
+                            Why a Bipartisan Revolt Killed the House's Spy Bill
+                          </a>
+                        </h4>
+                      </article>
+                    </div>
+                    <div class="col-md-4">
+                      <article>
```

Comment:
- [ ] For some reason this section is not displayed for medium or small devices. Kindly check that.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83211318-68f3f580-a122-11ea-8769-1bd2f29eeafb.png) | ![image](https://user-images.githubusercontent.com/57414789/83211286-54aff880-a122-11ea-94ab-5ad32c781335.png)